File: Number of Senior Citizens.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        count = 0
        for detail in details:
            age = int(detail[11] + detail[12])
            count += int(age > 60)
        return count

# class Solution:
#     def countSeniors(self, details: List[str]) -> int:
#         count = 0
#         for detail in details:
#             count += bool(int(detail[11] + detail[12]) > 60)
#         return count

File: Number of Senior Citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int count = 0;
        for (string& detail: details) {
            string c{detail[11], detail[12]};
            count += int(stoi(c) > 60);
        }
        return count;
    }
};
